### ISSUE TITLE
AnkoComponent is not found

### ISSUE THREAD

**Issue by vincaslt** [Kotlin/anko#113]
I am unable to use AnkoComponent in my project, I have imported `import org.jetbrains.anko.*` and

```
compile 'org.jetbrains.anko:anko-sdk15:0.7.3'
compile 'org.jetbrains.anko:anko-support-v4:0.7.3'
compile 'org.jetbrains.anko:anko-appcompat-v7:0.7.3'
```

is in my `build.gradle` file.

Writing `class ActivityUI : AnkoComponent { }` just highlights AnkoComponent in red and says it's unresolved.

**Comment by yanex** [comment]
`AnkoComponent` is not exist in Anko 0.7.3 and will be available since 0.8.

**Comment by vincaslt** [comment]
When is 0.8 being released and is there a way for me to use it already?
